src/aks-preview/HISTORY.rst

```diff
@@ -17,6 +17,11 @@ Pending
 ++++++++
 * [BREAKING CHANGE]: Remove support for `az aks get-os-options` command.
 
+6.0.0b2
```

Comment:
Please choose a new version number like 7.0.0b2

Reply:
And includes the content listed in section Pending

Reply:
done.